.github/workflows/ci.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         go test ./...
     # We set the maximum version of the go directive as 1.20 here
```

Comment:
```suggestion
    # We set the maximum version of the go directive as 1.21 here
```

Reply:
OK. Sorry. Maybe we should just remove these redundant information. To make future updates easier.